modules/backend/behaviors/FormController.php

```diff
@@ -232,6 +232,8 @@ public function create_onSave($context = null)
 
         $this->initForm($model);
 
+        $model->forceFill($this->formWidget->getSaveData());
```

Comment:
Why is this required?

Reply:
@LukeTowers  In case of creating new record, the model is initiated by [`$model = $this->controller->formCreateModelObject()`](https://github.com/octobercms/october/blob/master/modules/backend/behaviors/FormController.php#L230), at this point the model attributes are empty and when calling `Model::filterFields($attributes)` all attribute values are null. 

Calling `$model->forceFill($this->formWidget->getSaveData())` would fill the model with current form data and `$attributes` would have the actual values to be used inside `Model::filterFields`.

Reply:
@LukeTowers any chance we can get this reviewed soon?

Reply:
@MWalid I'm pretty busy right now, perhaps @w20k could take a look at it or you could offer a bounty on it to bump it up the priority list. Is forceFill how the model is populated in the update method?

Reply:
@MWalid This is a potential security issue allowing `forceFill` here, as that bypasses mass-assignment rules that may be implemented in the model. I don't think we can accept this PR with this in place.